sopel_modules/twitter/twitter.py

```diff
@@ -141,7 +141,7 @@ def format_time(bot, trigger, stamp):
         bot.db, bot.config, tz, trigger.nick, trigger.sender, parsed)
 
 
-@module.url(r'https?://twitter\.com/(?P<user>[^/]+)(?:$|/status/(?P<status>\d+)).*')
+@module.url(r'https?://(?:m(?:obile)\.)?twitter\.com/(?P<user>[^/]+)(?:$|/status/(?P<status>\d+)).*')
 @module.url(r'https?://twitter\.com/i/web/status/(?P<status>\d+).*')
```

Comment:
Turns out Twitter is perfectly happy to handle `/i/web/status` links on its mobile subdomains too.

```suggestion
@module.url(r'https?://(?:m(?:obile)\.)?twitter\.com/i/web/status/(?P<status>\d+).*')
```